packages/@react-spectrum/overlays/src/Popover.tsx

```diff
@@ -64,6 +73,8 @@ function Popover(props: PopoverProps, ref: DOMRef<HTMLDivElement>) {
   );
 }
 
+let ROOT_2 = Math.sqrt(2);
```

Comment:
btw, just learned this exists `Math.SQRT2` so we don't need to calculate it at all